src/screens/Services/Services.js

```diff
@@ -142,6 +132,46 @@ class ServicesScreen extends React.Component<Props> {
     ];
   };
 
+  getBuyCryptoServices = () => {
+    const {
+      user: { email = '' },
+      accounts,
+    } = this.props;
+
+    const address = getActiveAccountAddress(accounts);
```

Comment:
Just a thought: if we're about to drop KW soon, maybe we should enforce all fiat-bought tokens to settle on SW? Any thoughts on this @IAmKio  ?

Reply:
I agree, Smart Wallet only going forward - if they haven't activated their smart wallet yet then they should be prompted to do that first.